airflow/providers/docker/operators/docker.py

```diff
@@ -283,6 +283,8 @@ def __init__(
         self.dns = dns
         self.dns_search = dns_search
         self.docker_url = docker_url or os.environ.get("DOCKER_HOST") or "unix://var/run/docker.sock"
+        if not isinstance(self.docker_url, list):
+            self.docker_url = [self.docker_url]
```

Comment:
Better move convert into the hook, ho knows maybe one day `docker_url` become a templated field and hook called after the templates rendered